common/utilities/number/stabilized-value.h

```diff
@@ -6,6 +6,7 @@
 #include <string>
 #include <atomic>
 #include <mutex>
+#include <deque>
```

Comment:
This was discussed with Nir. For some reason deque was recognized in this file without an include, but when I removed the CATCH_CONFIG_MAIN flag it stopped recognizing it. As it is probably best to include it anyway that's what I decided to do.